src/main/java/core/basesyntax/FigureBehavior.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public abstract interface FigureBehavior {
+    public abstract double getArea();
+
+    abstract void drawFigure();
```

Comment:
Move these methods to separate interfaces

Reply:
And remove public and abstract words, it's redundant

Reply:
done